apps/docs/assets/scripts/theme-handler.js

```diff
@@ -42,6 +42,11 @@ button.addEventListener('click', toggleMenu);
 const menu = document.querySelector('vwc-menu#dark-mode-menu');
 menu.addEventListener('click', event => setTheme(event.target.text?.toLowerCase()));
 
+menu.addEventListener('keydown', event => {
+    if (event.key === 'Enter' || event.key === 'Space')
+        return setTheme(event.target.text?.toLowerCase());
+});
+
```

Comment:
Should not be necessary to do this
```suggestion
menu.addEventListener('change', event => {
  if (event.target.checked)
    setTheme(event.target.text?.toLowerCase()));
});
```